PSKoans/Koans/Constructs and Patterns/AboutRegularExpressions.Koans.ps1

```diff
@@ -577,7 +579,9 @@ Describe 'Meditative Examples' {
             )
         }
 
-        It 'validates user input' {            $validPhoneNumbers = [regex]::Matches($sanitizedNumbers, '____').Value
+        It 'validates user input' {            
+        
+        	$validPhoneNumbers = [regex]::Matches($sanitizedNumbers, '____').Value
```

Comment:
Hey, @udubnate! Thanks for your PR 😁 .

This is a nit, but I know @vexx32 is a perfectionist - the indentation on this line is out of whack.

Reply:
should be fixed now but @thomasrayner if you want to take another look before resolving conversation

Reply:
Heh, I think it's actually got to go the other way 😁 

![image](https://user-images.githubusercontent.com/13203965/81981854-bbdb9200-95e5-11ea-9598-82e2dcdff9af.png)